index.html

```diff
@@ -1,63 +1,66 @@
+    <link rel = "stylesheet" type= "text/css" href = "styles/normalize.css"/>
+    <link rel = "stylesheet" type= "text/css" href = "styles/skeleton.css"/>
     <link rel = "stylesheet" type= "text/css" href = "styles/style.css"/>
-
     <title>Mike's Blog</title>
   </head>
 
+
+<body>
   <header>
     <h1> Mike's Blog</h1>
     <nav>
-      <ul>Navigation: </br>
-        <li>Home</li>
-        <li>About</li>
+      <ul>
+        <li><a href="#" id="home">Home</a></li>
+        <li><a href="#" id="about">About</a></li>
+        <li><a href="https://github.com/Mikekh84">Github</a></li>
+        <li><a href="https://twitter.com/CSS_Ipsum">Twitter</a></li>
       </ul>
-
-
     </nav>
   </header>
 
-  <body>
-
 
-    <main>
+  <main>
+    <section id="bio">
+      <p>This is my code. There are many like it, but this one is mine.
+        My code is my best friend. It is my life. I must master it as I must master my life.<p>My code, without me, is useless. Without my code, I am useless. I must fire my code true. I must shoot straighter than my enemy who is trying to kill me. I must shoot him before he shoots me. I will...</p>
+        <p>My code and I know that what counts in war is not the rounds we fire, the noise of our burst, nor the smoke we make. We know that it is the hits that count. We will hit...</p>
+        <p>My code is human, even as I, because it is my life. Thus, I will learn it as a brother. I will learn its weaknesses, its strength, its parts, its accessories, its sights and its barrel. I will keep my code clean and ready, even as I am clean and ready. We will become part of each other. We will...</p>
+        <p>I swear this creed. My code and I are the defenders of my country. We are the masters of our enemy. We are the saviors of my life.
+        So be it, until victory is America's and there is no enemy, but peace!</p>
+    </section>
 
-
-      <form>
-        Filters<br/>
+      <form>Filters<br/>
         <select id="authFilter">
-          <option>Author</option>
+          <option class="author" id="dropDownAuthor">Author</option>
         </select>
-
         <select id="catFilter">
-          <option class="category">Category</option>
+          <option class="category" id="dropDownCat">Category</option>
         </select>
       </form>
 
-        <article class="template">
+      <article id="template" class="template">
           <h1 class="heading"></h1>
           <div class="author"></div>
-          <div class="url"></div>
-          <div class="artbody"></div>
           <div class="pub"></div>
+          <div class="artbody"></div>
           <div class="cat"></div>
-<<<<<<< HEAD
       </article>
-    </main>
-
-=======
-        </article>
-      </main>
-      <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.0.0/highlight.min.js"></script>
->>>>>>> master
-      <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```

Comment:
is this working? looks like you've got a github conflict. (but I am not seeing errors in the page). make sure to clean this in your master version if you haven't already